testdata/go/v2/memcached-operator/Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY api/ api/
 COPY controllers/ controllers/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
```

Comment:
I noticed that there was a [commit](https://github.com/kubernetes-sigs/kubebuilder/commit/c9f63e8147a956547d7574f9abea0a3569d720df) that added the `GO111MODULE=on` back to the Dockerfile scaffold template for go/v2. Will this difference cause any issues for users wanting to scaffold with go/v2?